CMakeLists.txt

```diff
@@ -418,8 +424,47 @@ if (WITH_ROS1_INTERFACE AND catkin_FOUND AND CATKIN_DEVEL_PREFIX)
     install(TARGETS knowrob-ros RUNTIME DESTINATION bin)
 
 
+
+        
+
+	## ROS Client Interface
+	add_library( 
+		knowrob-ros-cli
+
+		include/knowrob/ros_client/cpp/knowrob_client.hpp
+		src/ros_client/knowrob_client.cpp	
+	)
+
+	add_dependencies(
+		knowrob-ros-cli
+		${catkin_EXPORTED_TARGETS}
+	)
+
+	target_link_libraries(
+		knowrob-ros-cli
+		${catkin_LIBRARIES}
+	)
+
+	install(
+  		TARGETS knowrob-ros-cli
+  		LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
+	)
+
+	## Test node for the client library
+	add_executable(
+		client-node
+		client-node
```

Comment:
This doesn't seem to build for me, i think the path to the cpp is missing?

Reply:
i will investigate when i move the client node